eval/final/components.py

```diff
@@ -20,10 +18,13 @@ def run_final_eval_op(
     candidate_model: str = None,
     taxonomy_path: str = "/input/taxonomy",
     sdg_path: str = "/input/sdg",
+    mmlu_branch_output_path: str = "/output/mmlu_branch",
+    mt_bench_branch_output_path: str = "/output/mt_bench_branch",
```

Comment:
This change means we no longer output this file(s) as artifacts. We only store the data on PVC and generate metrics from it. But we no longer upload it as `Output[Artifact]` Is this intentionally replaced with the metrics?

If yes, we should also remove `output_mt_bench_task`
https://github.com/opendatahub-io/ilab-on-ocp/pull/241/files#diff-6b92e3e77956f8020482ca91889a24f3411c9a4854f2aaaac30ee504ef26ede4R443-R445 for consistency (uploading `mt_bench` only JSON and omitting `mmlu_branch` and `mt_bench_branch` makes no sense).

If no, we should introduce similar upload step(s) as `output_mt_bench_task` for `mmlu_branch` and `mt_bench_branch` or join those with `output_mt_bench_task` into a single step...

Reply:
Yes, I think we originally pulled the metrics from the Artifacts, but then decided to use PVC's instead for consistency and speed. However, I think I replaced the Artifact steps with PVCs instead of adding it. But, I think it makes sense to maintain the eval steps' outputs as Artifacts too. Good catch. I will update the PR to make sure the final eval outputs are stored as artifacts. Thanks 😄

Reply:
I've added `pvc_to_mt_bench_branch_op` and `pvc_to_mmlu_bench_op` to the `utils/components.py` and added them to the pipeline to store the final eval outputs as artifacts.